public/pages/pages/hiyori/hiyori.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+        <title>hiyori</title>
+        <script src="https://unpkg.com/alpinejs" defer></script>
+    </head>
+    <body>
+        <div x-data="hiyoriCtrl">
+            <div class="name">
+                <span>name: </span>
+                <span x-text="name"></span>
+            </div>
+            <div class="age">
+                <span>age: </span>
+                <span x-text="age"></span>
+            </div>
+        </div>
+        <script>
+            function hiyoriCtrl() {
+                return{
+                    name: 'hiyori',
+                    age: 18,
+                }
+            }
+        </script>
+    </body>
+</html>
```

Comment:
```suggestion
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
    <title>hiyori</title>
    <script src="https://unpkg.com/alpinejs" defer></script>
  </head>
  <body>
    <div x-data="hiyoriCtrl">
      <div class="name">
        <span>name: </span>
        <span x-text="name"></span>
      </div>
      <div class="age">
        <span>age: </span>
        <span x-text="age"></span>
      </div>
    </div>
    <script>
      function hiyoriCtrl() {
        return{
            name: 'hiyori',
            age: 18,
        }
      }
    </script>
  </body>
</html>
```

インデントをスペース2つにしてほしいです :pray: